src/plugins/intel_cpu/src/mkldnn_graph_optimizer.cpp

```diff
@@ -1117,17 +1117,29 @@ void MKLDNNGraphOptimizer::FuseConvolutionSumAndConvolutionSumActivation(MKLDNNG
             isSuitableParent2 = isSuitableParent2 && canFuseSum(binConvNode2, graphNode);
         }
 
+        auto checkFusedWith = [](MKLDNNConvolutionNode* conv) -> bool {
+            for (const auto& node : conv->getFusedWith()) {
+                if (node->getAlgorithm() == EltwiseAdd)
```

Comment:
you also need to check that it's `isSpecialConvolutionAddFusing()`, because fusing with `EltwiseAdd` also possible when we fuse add as ScaleShift

Reply:
Good catch, thanks!